alea/submitters/local.py

```diff
@@ -226,7 +238,7 @@ def submit(
                         "threshold": [],
                         "poi_expectation": [],
                     }
-                    threshold[threshold_key] = threshold_value
+                    threshold[threshold_key] = deepcopy(threshold_value)
```

Comment:
Very important because the shallow copy will convey the modification of `nominal_values` and `generate_values` to `threshold_value` and `threshold`.